python/nano/src/bigdl/nano/pytorch/encryption/torch/dispatcher.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO: consider move this to nano's dispatcher
+
+_torch_patch = None
+is_torch_patched = False
+
+
+def patch_torch():
```

Comment:
should we just call it patch_encryption and expose it to https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/pytorch/patching/__init__.py?

Reply:
patch_torch is now in an awkward situation since we have not decided to make it an unified patching API for users, while `bigdl.nano.pytorch.patching` will include all the specific patch function we want to expose to our users.

Reply:
Yeah, I agree.  

Will rename it to `patch_encryption` and move it to `bigdl.nano.pytorch.patching`